README.md

```diff
@@ -1,3 +1,5 @@
+[![MseeP.ai Security Assessment Badge](https://mseep.net/pr/devopness-devopness-badge.png)](https://mseep.ai/app/devopness-devopness)
```

Comment:
@lwsinclair could you please update the assessment on https://mseep.ai/app/devopness-devopness ?

I beileve all those vulnerabilities have been already addressed by recent changes in this repo.